packages/models-library/src/models_library/basic_types.py

```diff
@@ -21,7 +21,7 @@
 EnvVarKey = constr(regex=r"[a-zA-Z][a-azA-Z0-9_]*")
 
 # e.g. '5c833a78-1af3-43a7-9ed7-6a63b188f4d8'
-UUIDStr = constr(regex=UUID_RE)
+UUIDStr = constr(regex=UUID_RE_BASE)
```

Comment:
MINOR: I wonder if ``constr(strip_whitespace=True, regex=UUID_RE)`` would solve this? 

SEE https://pydantic-docs.helpmanual.io/usage/types/#arguments-to-constr

Reply:
ah I am not sure what you refer to. Actually I think I should keep it with UUID_RE, will double-check.

Reply:
I mean the difference between BASE and non-BASE is the regex "^" (starts with). I wonder if you strip the strings, you will always have the value that you search at the start of the string, right?